### PR TITLE
[no ticket][risk-no] Delete few e2e tests

### PR DESCRIPTION
**Pull request by yonghaoy**
- notebook-ui.spec: Removed, because it is covered by UI unit test
- workspace-ui.spec: Removed. See https://docs.google.com/spreadsheets/d/16DatY69Z7BQXAHfdaRcwZJImMPmSrz0NDcPyms1dkAM/edit?resourcekey=0-_rjuBpZiS4NcqCU26McMTQ#gid=0
- lock workspace. Best effort to delete workspace after test. We have nightly cron job can delete that workspace. 

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: e2e/tests/nightly/admin/lock-workspace.spec.ts [hunk 1]
```diff
@@ -124,6 +125,11 @@ describe('Workspace Admin lock-workspace', () => {
     await workspaceEdit.waitForLoad();
     await workspaceEdit.clickCancelButton();
     // cleanup
-    await dataPage.deleteWorkspace();
+    try {
+      await dataPage.deleteWorkspace();
```
Comment (NehaBroad) [inline e2e/tests/nightly/admin/lock-workspace.spec.ts:129]: Why cant we delete the workspace in after block?

File: e2e/tests/ui/workspace-ui.spec.ts [hunk 2]
```diff
@@ -1,189 +0,0 @@
...
-        const workspaceAccessLevel = await card.getAccessLevel();
-        expect(workspaceAccessLevel).toEqual(accessLevel.toUpperCase());
-        // Verify: Share, Edit, Duplicate and Delete actions.
-        await card.verifyWorkspaceCardMenuOptions(workspaceAccessLevel);
-        break;
-      }
-    }
-  });
-
-  test('Duplicate Workspace page', async () => {
```
Comment (NehaBroad) [inline e2e/tests/ui/workspace-ui.spec.ts:137]: Curious if we are testing actual duplicate workspace flow anywhere

File: e2e/tests/nightly/admin/lock-workspace.spec.ts (hunk 1 above)
Comment (yonghaoy) [reply e2e/tests/nightly/admin/lock-workspace.spec.ts:129]: Sometimes workspace fail to delete due to various issues, like Terra issue or some APP test uses that workspace but not deleting that.

File: e2e/tests/ui/workspace-ui.spec.ts (hunk 2 above)
Comment (yonghaoy) [reply e2e/tests/ui/workspace-ui.spec.ts:137]: I think we have some tests to cover various cases of duplicate workspace: https://github.com/all-of-us/workbench/blob/c60057bc5e9b5e0b61559e87d9b74b20186965db/ui/src/app/pages/workspace/workspace-edit.spec.tsx#L186

File: e2e/tests/nightly/admin/lock-workspace.spec.ts (hunk 1 above)
Comment (NehaBroad) [reply e2e/tests/nightly/admin/lock-workspace.spec.ts:129]: My only concern is if we run it on local, it leaves behind workspaces which could be distracting

**Review by NehaBroad — APPROVED** [review]

**Comment by yonghaoy** [comment]
I think our cronJob https://github.com/all-of-us/workbench/blob/3a64c43b39a09f6f595cd9b3ab034917f92a2b59/api/src/main/java/org/pmiops/workbench/api/OfflineTestUsersController.java#L77 should eventually catch up (but we need to use e2e test user account)

File: e2e/tests/ui/workspace-ui.spec.ts (hunk 2 above)
Comment (Peter-Lavigne) [reply e2e/tests/ui/workspace-ui.spec.ts:137]: This functionality is covered by another E2E test: `workspace-duplicate.spec.ts`, so I don't think we're losing any coverage by removing this one.

**Review by Peter-Lavigne — APPROVED** [review]
Thanks for doing this. I agree that the scope of these tests could be covered by our UI unit tests or are already covered by other E2E tests. I didn't do a deep dive into each test but I trust your judgement here.

**Review by jmthibault79 — APPROVED** [review]

File: e2e/tests/nightly/admin/lock-workspace.spec.ts [hunk 3]
```diff
@@ -124,6 +125,11 @@ describe('Workspace Admin lock-workspace', () => {
     await workspaceEdit.waitForLoad();
     await workspaceEdit.clickCancelButton();
     // cleanup
-    await dataPage.deleteWorkspace();
+    try {
```
Comment (dmohs) [inline e2e/tests/nightly/admin/lock-workspace.spec.ts:128]: Alternatively:
```javascript
dataPage.deleteWorkspace().catch(() => logger.warn('fail to delete workspace'))
```

File: e2e/tests/nightly/admin/lock-workspace.spec.ts (hunk 1 above)
Comment (dmohs) [reply e2e/tests/nightly/admin/lock-workspace.spec.ts:129]: Given the above conversation, this wants a comment or Joel's suggestion. Something like:
```javascript
// Attempt to clean up here, reducing the load on the nightly cron job.
```
